docs/mcp/client.md

```diff
@@ -29,7 +29,7 @@ Examples of both are shown below; [mcp-run-python](run-python.md) is used as the
 [`MCPServerSSE`][pydantic_ai.mcp.MCPServerSSE] connects over HTTP using the [HTTP + Server Sent Events transport](https://spec.modelcontextprotocol.io/specification/2024-11-05/basic/transports/#http-with-sse) to a server.
 
 !!! note
-    [`MCPServerSSE`][pydantic_ai.mcp.MCPServerSSE] requires an MCP server to be running and accepting HTTP connections before calling [`agent.run_mcp_servers()`][pydantic_ai.Agent.run_mcp_servers]. Running the server is not managed by PydanticAI.
+    [`MCPServerSSE`][pydantic_ai.mcp.MCPServerSSE] requires an MCP server to be running and accepting HTTP connections before calling [`agent.run_toolsets()`][pydantic_ai.Agent.run_toolsets]. Running the server is not managed by PydanticAI.
```

Comment:
Seems a bit odd to me to assume toolsets need to be run. I can see why MCP servers need to be run, but my feeling is that it should be the MCP server that gets run rather than some generalized method call on the agent, when most toolsets will either have custom run logic or none at all. 🤷‍♂️ maybe I'll appreciate it more once I've read more..

Reply:
I agree, I think I like `async with agent:` better, as we could do all kinds of things in there not just "run (MCP) toolsets".